content/sensu-go/5.19/sensuctl/reference.md

```diff
@@ -648,6 +648,109 @@ See the [RBAC reference][21] for information about using access control with nam
 
 See the [RBAC reference][22] for information about local user management with sensuctl.
 
+#### sensuctl prune
+
+{{% notice important %}}
+**IMPORTANT**: `sensuctl prune` is an alpha feature in release 5.19.0 and may include breaking changes.
+{{% /notice %}}
+
+**COMMERCIAL FEATURE**: Access sensuctl pruning in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][30].
+
+The `sensuctl prune` subcommand allows you to delete resources that do not appear in a given set of Sensu objects (called a "configuration") from a from a file, URL, or STDIN.
+For example, you can use `sensuctl create` to to apply a new configuration, then use `sensuctl prune` to prune unneeded resources, resources that were created by a specific user or that include a specific label selector, and more.
+
+`sensuctl prune` can only delete resources that have the label `sensu.io/managed_by: sensuctl`, which Sensu automatically adds to all resources created with sensuctl.
+This means you can only use `sensuctl prune` for delete resources that were created with sensuctl.
+
+The pruning operation always follows the role-based access control (RBAC) permissions of the current user.
+For example, to prune resources in the `dev` namespace, the current user who sends the prune command must have delete access to the `dev` namespace.
+
+##### sensuctl prune usage
+
+{{< highlight shell >}}
+sensuctl prune [RESOURCE TYPE],[RESOURCE TYPE]... [-f [FILE] -f [URL] -f [STDIN] [-r] ... ] [--NAMESPACE] [flags]
+{{< /highlight >}}
+
+In this example `sensuctl prune` command:
+
+- Replace [RESOURCE TYPE] with the [synonym or fully qualified name][48] of the resource you want to prune.
+You must specify at least one resource type or the `all` qualifier (to prune all resource types).
+- Replace [FILE], [URL], or [STDIN] with the name of the file, URL, or STDIN where you want to apply the pruning.
+- Replace [flags] with the flags you want to use, if any.
+- Replace [--NAMESPACE] with the namespace where you want to apply pruning.
+To prune all namespaces, omit the namespace qualifier from your command.
```

Comment:
Sensuctl does not currently support pruning from multiple namespaces. Users should issue multiple `sensuctl prune` commands in each namespace they want to prune resources from. There is work to enhance this with a `--all-namespaces` and/or `--namespaces` flag.